cocotb/_build_libs.py

```diff
@@ -188,6 +194,10 @@ def _get_python_lib():
 # Make PRI* format macros available with C++11 compiler but older libc, e.g. on RHEL6.
 _extra_cxx_compile_args += ["-D__STDC_FORMAT_MACROS"]
 
+# enable 32bit cross compilation
+if platform.architecture()[0] == "32bit":
+    _extra_cxx_compile_args += ["-m32"]
```

Comment:
Again, this doesn't make sense to me

Reply:
I think python assumes gcc is 32bit (than you do not need -m32).

Reply:
I'd appreciate it if you could find a reference for that in the python bugtracker.

Also, I think " support systems with 32-bit python but 64-bit gcc" would be more helpful in the comment

Reply:
Only what I could find:
https://stackoverflow.com/questions/10478802/compiling-32-bit-python-modules-e-g-numpy-on-64-bit-linux

This can be different for different platforms potentially. Anaconda 32bit does not set "-m32" by default.

Reply:
Can always use `CFLAGS="-m32" pip install ..` but I thought this can be automatic.

Reply:
> Anaconda 32bit does not set "-m32" by default.

Doesn't anaconda 32-bit come bundled with a 32-bit GCC?

Reply:
Not by default (for me). Not sure how this looks if one compiles own python. 

Should better remove this?

Reply:
Yeah, I think I'd prefer it for now. Can put up as a separate patch, but I'd like a bugs.python.org reference before we put in a workaround - even if that means filing an issue ourselves.

Reply:
I have cross-compiled python on centos 7 and -m32 is added automatically in this case, so it looks like this is conda. Our instructions recommend conda+gcc so should be fine: https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions#32-bit-python

Sorry for the confusion.